### ISSUE TITLE
Magic category improvement

### ISSUE THREAD

**Issue by KoLeS2109** [Cvolton/GMDprivateServer#931]
Does anyone have any suggestions on how to improve the magic category algorithm? For example choose levels that have a lot of downloads & level's age <= 7 days.

![Screenshot_20220928-185704_ES File Explorer.jpg](https://user-images.githubusercontent.com/93978604/192827994-8ea5fbf4-c9cd-4d3f-9427-d3cb56867e56.jpg)

**Comment by MegaSa1nt** [comment]
ща я попробую у себя, потом тебе напишу

**Comment by KoLeS2109** [comment]
Да, можно реализовать отбор уровней модераторами во вкладку magic, но можно и сделать старый алгоритм отбора уровней, чтобы они вместе работали, если это возможно

**Comment by MegaSa1nt** [comment]
> Да, можно реализовать отбор уровней модераторами во вкладку magic, но можно и сделать старый алгоритм отбора уровней, чтобы они вместе работали, если это возможно

Ща попробую у себя
